Add notes for `fileStat`

satotake opened the pull request:
Notes for https://github.com/gohugoio/hugo/pull/5036

satotake commented:
Fixed function name `filseStat` => `os.Stat` in response to  https://github.com/gohugoio/hugo/pull/5036

kaushalmodi commented on content/en/functions/fileStat.md:
````diff
@@ -0,0 +1,33 @@
+```
+
+Function [`fileStat`][fileStat] returns [`os.FileInfo`][osfileinfo].
+For Further information of `os.FileInfo`, see  [golang page][osfileinfo].
````
Lower case "further" in "For Further".

kaushalmodi commented on content/en/functions/os.Stat.md:
````diff
@@ -0,0 +1,33 @@
+{{ $stat.Size }} → 30
+```
+
+Feunction [`os.Stat`][Stat] returns [`os.FileInfo`][osfileinfo].
````
Fix typo

kaushalmodi commented on content/en/functions/os.Stat.md:
````diff
@@ -0,0 +1,33 @@
+```
+
+Feunction [`os.Stat`][Stat] returns [`os.FileInfo`][osfileinfo].
+For Further information of `os.FileInfo`, see [golang page][osfileinfo].
````
Same as before.. lower case "further".

satotake commented:
@kaushalmodi Thank you for your comment.
I fixed them. In addition, I removed `content/en/functions/fileStat.md` because `fileStat` was renamed `os.Stat` finally.
I am sorry for my carelessness and late response.
